### PR TITLE
Add profile for GCP CloudSql Postgres driver

### PR DESCRIPTION
**Pull request by jakubadamek**

### CODE REVIEW

**Comment by jkiddo** [comment]
Please add a comment in the poem that this profile also needs eg. the `boot` profile to work

**Comment by jakubadamek** [comment]
> Please add a comment in the poem that this profile also needs eg. the `boot` profile to work

Thanks, done

**Comment by dotasek** [comment]
Is there a way to test what this new dependency adds to the functionality? (example configs, start params, etc.?)

File: pom.xml [hunk 1]
```diff
@@ -676,5 +676,16 @@
                 </plugins>
             </build>
         </profile>
+        <!-- For connecting to GCP CloudSQL Postgres instances. Needs 'boot' profile as well. -->
```
Comment (jkiddo) [inline pom.xml:679]: You might consider putting the link you found here as well (the https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/main/docs/jdbc.md#postgres-1 one)

Comment (jakubadamek) [reply pom.xml]: Done

**Review by jkiddo — APPROVED** [review]

**Pull request merged**